packages/language/src/language-server/completion/follow-elements.ts

```diff
@@ -98,6 +95,22 @@ function getFollowElementsForUnknownToken(token: Token): FollowElement[] {
     ];
   }
 
+  const categories = token.tokenType.CATEGORIES?.values() ?? [];
+
+  /**
+   * We are maybe inside a declaration which is not followed by other statements, e.g.:
+   *
+   * `DCL A `
+   */
+  if (categories.some((v) => v.name === "ID")) {
+    return [
+      {
+        kind: FollowKind.CstNode,
+        types: DataSpecificationCompletionKeywordsArray,
+      },
+    ];
+  }
```

Comment:
Suggestion: I believe we can ignore this for now if the token hasn't been successfully parsed. Given the abundance of `ID`-like tokens (essentially every word), we should be rather careful what we present in the completion in case the parsing hasn't encountered this token.